omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonNameResource1_8.java

```diff
@@ -217,6 +217,14 @@ public void purge(PersonName pn, RequestContext context) throws ResponseExceptio
 	@Override
 	public PersonName save(PersonName newName) {
 		// make sure that the name has actually been added to the person
```

Comment:
Shouldn't this comment be moved to after the if block? I think it's misleading in a way. What do you think??

Reply:
Hi @trevor-james-nangosha 
Good point. 

Thanks